src/Compilers/VisualBasic/Portable/Errors/MessageProvider.vb

```diff
@@ -594,6 +594,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Public Overrides ReadOnly Property ERR_DataSectionStringLiteralHashCollision As Integer
```

Comment:
>ERR_DataSectionStringLiteralHashCollision [](http://example.com/codeflow?start=43&length=41)

Are we testing this from VB?

Reply:
This cannot be hit from VB, the feature is not implemented there (yet?). I guess I can change this to throw Unreachable exception then.